src/index.html

```diff
@@ -13,20 +13,34 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="styles/style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
```

Comment:
The class name 'page' does not represent the meaning of the content. Consider using a class name that reflects the role or content of the `body` element.